src/app/src/pages/ForgotPassword.js

```diff
@@ -0,0 +1,88 @@
+import { useState } from 'react';
+import { Box, Button, Input, Text, VStack } from '@chakra-ui/react';
+import { useNavigate } from 'react-router-dom';
+
+import { API } from '../api';
+import { API_URLS } from '../constants';
+
+export default function ForgotPassword() {
+    const [email, setEmail] = useState('');
+    const [success, setSuccess] = useState(false);
+    const [errorDetail, setErrorDetail] = useState('');
+    const navigate = useNavigate();
+
+    const formatApiError = apiErrorData => {
+        var errorDetail = '';
+        Object.values(apiErrorData).forEach(val => (errorDetail += `${val}`));
+        return errorDetail || 'An unknown error occurred.';
+    };
+
+    const forgotPasswordRequest = () => {
+        API.post(API_URLS.FORGOT, { email })
+            .then(() => {
+                setSuccess(true);
+                setErrorDetail('');
+            })
+            .catch(apiError => {
+                setSuccess(false);
+                setErrorDetail(formatApiError(apiError.response?.data));
+            });
+    };
+
+    return (
+        <VStack bg='gray.100' h='100vh' spacing={5}>
+            <Box h='150px'></Box>
+            {success ? (
+                <>
+                    <Text textStyle='forgotPasswordHeader'>
+                        If an account exists for {email}, instructions for
+                        resetting your password have been sent.
+                    </Text>
+                    <Button variant='cta' onClick={() => navigate('/login')}>
+                        Login
+                    </Button>
+                </>
+            ) : (
+                <>
+                    {}
```

Comment:
What is this for?

Reply:
auto-formatting gone haywire! good spot. 👍🏻